kotkt.nl/content/_index.nl.md

```diff
@@ -60,12 +60,12 @@ Wil je de sfeer proeven? Kom dan vooral een keer op onze bord- of rollenspelavon
 {{< aside >}}
    {{< banner rotate="6" padding="41px">}}
        {{< figure src="images/Rollenspellen_icons.png" >}}
-       One-shot rollenspelavond
-       Elke 1e woensdag van de maand
+       **One-shot rollenspelavond: **
+       Elke 1e woensdag van de maand  
        
-       Looking For Group
+       **Looking For Group: **
        Volgende avond:
-       TBA 
+       Woensdag 14 Oktober 2020 (19.30u)
```

Comment:
Misschien link naar Squire er in zetten?

Reply:
Dat zou ik juist niet doen omdat niet-leden (de grootste "doelgroep" van de website) zich sowieso moeten inschijven via een google form. Ik denk dat een link naar Squire juist verwarrend zal zijn.